frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -191,7 +184,9 @@ export const LandingPage = (): JSX.Element => {
         </Stack>
       </LandingSection>
       <LandingSection>
-        <SectionTitleText>Used by most government agencies</SectionTitleText>
+        <SectionTitleText>
+          Used by every Singapore government agency
```

Comment:
/justSuggestionCanIgnore
```suggestion
          Used by all Singapore government agencies
```